src/main/java/uk/co/ramp/covid/simulation/place/ConstructionSite.java

```diff
@@ -36,5 +36,9 @@ public Shifts getShifts() {
     public boolean isFullyStaffed() {
         return nStaff > 0;
     }
+    
+    public void updateTransProb() {
```

Comment:
Why do only construction sites have this function? Seems like all/most places should have it?

This function is also unused?